src/transaction/log_page_buffer.c

```diff
@@ -2380,10 +1929,11 @@ logpb_write_page_to_disk (THREAD_ENTRY * thread_p, LOG_PAGE * log_pgptr, LOG_PAG
 	}
 
       logpb_fatal_error (thread_p, true, ARG_FILE_LINE, "logpb_write_page_to_disk");
-      return NULL;
+      mnt_log_iowrites (thread_p, 1);
```

Comment:
@funkydvd 

Please move the statement to outside of `if` block. This is an error handling block. :)